src/main/java/org/mule/module/apikit/FlowFinder.java

```diff
@@ -187,9 +188,10 @@ private void logMissingMappings(String version) {
         }
         if (action.hasBody()) {
           for (String contentType : action.getBody().keySet()) {
-            if (restFlowMap.get(key + ":" + getMediaType(contentType)) == null) {
+            String mediaType = getMediaType(contentType);
```

Comment:
Handle UnsupportedMediaTypeException, this method only logs resources without flows mappings, we dont want to fail the application start up because of this